src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -119,6 +119,20 @@ public void start(Future<Void> startFuture) {
       .withRequiredProperties("name", "loanable", "renewable")
       .create().register(router);
 
+    new FakeStorageModuleBuilder()
+      .withRecordName("request policy")
+      .withRootPath("/request-policy-storage/request-policies")
+      .withCollectionPropertyName("requestPolicies")
+      .withRequiredProperties("name", "allowHold", "allowPage", "allowRecal")
```

Comment:
Similarly to the builder, please can this be changed to match the request policy record structure described by [request-policy-storage](https://github.com/folio-org/mod-circulation-storage/blob/master/ramls/request-policy.json)

Reply:
Corrected in https://github.com/folio-org/mod-circulation/pull/182/commits/05a744fb8a2c8205bf74786c98bb0a41cd18cf08